docs/developer-guide/plugins.md

```diff
@@ -62,10 +62,9 @@ There are examples of plugins providing simple examples of how to:
 * perform an action response object  - `loadtest.plugins.AsyncResponseContentDecoderPluginFactory`
 
 ### Plugin class
-A Styx plugin must implement a Plugin interface, which extends from HttpInterceptor 
-interface. Thus every Styx plugin is an HttpInterceptor. As the name suggests, 
-HttpInterceptor(s) can intercept and transform, or perform some other action, HTTP 
-traffic as it is being proxied through. The interceptors are organised linearly in 
+A Styx plugin must implement a `Plugin` interface, which extends from `HttpInterceptor`. 
+As name suggests, an HttpInterceptor intercepts, transforms, or performs other actions, 
```

Comment:
"As name suggests" -> "As the name suggests"

"other actions, " -> "other actions "